pytorch_lightning/utilities/deprecation.py

```diff
@@ -34,37 +34,41 @@ def get_func_arguments_and_types(func: Callable) -> List[Tuple[str, Tuple, Any]]
     return name_type_default
 
 
-def deprecated(target: Callable, ver_deprecate: str = "", ver_remove: str = "") -> Callable:
+def deprecated(target: Callable, ver_deprecate: Optional[str] = "", ver_remove: Optional[str] = "") -> Callable:
```

Comment:
```suggestion
def deprecated(target: Callable, ver_deprecate: str = "", ver_remove: str = "") -> Callable:
```
Optional[...] references to Union[..., None]. Here, the default is "".